test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+
+        // Whenever a row is selected, confirm that it is the only row that appears selected
+        this.checkRows(rowNum);
+        return cy.get('@row');
+    },
+    clickAddButton() {
+        cy.getByDataPreview('ContentExplorer')
+            .find('.be-btn-add')
+            .click();
+    },
+    clickUploadButton() {
+        cy.get('.dropdown-menu-element > ul > li')
+            .contains('Upload')
+            .click();
+    },
+    clickNewFolderButton() {
+        cy.get('.dropdown-menu-element > ul > li')
+            .contains('New Folder')
+            .click();
+    },
+    clickNewFolderCancelButton() {
+        cy.get('.be-modal-btns > button')
+            .contains('Cancel')
+            .click();
+    },
+    clickNewFolderCreateButton() {
+        cy.get('.be-modal-btns > button')
+            .contains('Create')
+            .click();
+    },
+    clickCloseUploadModal() {
+        cy.get('.bcu-footer-left > button')
+            .contains('Close')
+            .click();
+    },
+    clickShareButton(rowNum) {
+        this.getRow(rowNum)
+            .find('.bce-more-options')
+            .contains('Share')
+            .click();
+    },
+    clickCloseShareButton() {
+        cy.get('.be-modal-btns > button')
+            .contains('Close')
+            .click();
+    },
+    clickMoreOptionsButton(rowNum) {
+        this.getRow(rowNum)
+            .find('.bce-btn-more-options')
+            .click();
+    },
+    clickRenameButton() {
+        cy.get('.dropdown-menu-element')
+            .contains('Rename')
+            .click();
+    },
+    clickCloseRenameButton() {
+        cy.get('.be-modal-btns > button')
+            .contains('Cancel')
+            .click();
+    },
+    previewRow(rowNum) {
+        this.getRow(rowNum)
+            .find('.be-item-name > button')
+            .click();
+    },
+    closePreview() {
+        cy.get('.bcpr-btns')
+            .find('[aria-label="Close"]')
+            .click();
+    },
+};
+
+describe('ContentExplorer', () => {
+    beforeEach(() => {
+        cy.server();
+        cy.route('GET', '**/folders/*', 'fixture:content-explorer/root-folder.json');
+    });
+
+    describe('Selection', () => {
+        it('Should not have a selected row to start', () => {
+            helpers.load();
+            helpers.checkRows(-1);
+        });
+
+        it('Should be able to select a row', () => {
+            helpers.load();
+            helpers.selectRow(3);
+        });
+
+        it('Should change selected rows', () => {
+            helpers.load();
+            helpers.selectRow(3);
+            helpers.selectRow(5);
+        });
+
+        it('Should open and close upload modal', () => {
+            helpers.load();
+            helpers.selectRow(2);
+            helpers.clickAddButton();
+            helpers.clickUploadButton();
+            helpers.clickCloseUploadModal();
+            helpers.checkRows(2);
```

Comment:
It's unclear to me what these tests are doing from reading these lines. I think the assertions should live directly in the test rather than in helpers.